src/vocab-topic.ttl

```diff
@@ -13,7 +13,7 @@
 
 <https://ontology.okp4.space/thesaurus/topic/AgricultureEnvironmentAndForestry> a skos:Concept ;
   skos:inScheme <https://ontology.okp4.space/thesaurus/topic> ;
-  skos:prefLabel "Agriculture, environment and forestry"@en .
+  skos:prefLabel "Agriculture, food, environment and forestry"@en .
```

Comment:
If you add the word `food` in the label, shouldn't we change the vocabulary term to `AgricutureFoodEnvironmentAndForestry`?

Reply:
By the way, I'm wondering if it would be more beneficial to adhere to a standard vocabulary, such as [Inspire - metadata-codelist/TopicCategory](https://inspire.ec.europa.eu/metadata-codelist/TopicCategory), and [map](https://www.w3.org/TR/skos-reference/#mapping) the terms accordingly. This would provide a wider audience and better interoperability imho. This requires to "split" of terms and eventually assembling them into a [skos:Collection](https://www.w3.org/TR/skos-reference/#collections), so we have a hierarchical structure that would allow for better organization and navigation of the terms.

Your opinion @MdechampG @henigh.

Reply:
@ccamel I totally agree with you. I am not totally convinced of this example though. Maybe we could align to IDSA if they have a standard about that. 
Could you have a look on https://github.com/International-Data-Spaces-Association/InformationModel ?

Reply:
@ccamel I think that INSPIRE Project only covers geo-referenced data in the EU (e.g., [Copernicus Open Access Hub](https://scihub.copernicus.eu/))
And I have a feeling that it is more used in governments and institutions, more than in Business context.   

We have been in contact (and member) of [Aggatway](https://www.aggateway.org/) that provides standards and might be helpful (they produce, open-access papers and documentation)

Regarding this subject, my suggestion is to benchmark a "high-level" standards vocabulary that covers larger topics, like Business categories, and sub-categories in the world, and not only agriculture.

Maybe it will be challenging to find one, but it is important to significantly improve interoperability. 
cc @MdechampG

Reply:
Yes I agree with you @henigh to have high-level vocabulary. Aggateway is focus on Agriculture only so it is not relevant for topics I think. 
Maybe we can have a look here : https://ddialliance.org/project/the-dataverse-network but it's about research and not business.

Reply:
And another link to dig in : http://iridl.ldeo.columbia.edu/ontologies/

Reply:
@MdechampG For https://github.com/International-Data-Spaces-Association/InformationModel, there is no taxonomy found in the project that expresses the concept of "topic".